.github/workflows/gh-ci.yaml

```
@@ -150,7 +150,7 @@ jobs:
 
     - name: Build package
       run: |
-        python setup.py sdist
+        python -m pip install .
```

```suggestion
        python3 -m pip install --upgrade build
```

This command used to build a tarball (`sdist`) for distribution, I believe this should be the new syntax.